src/internal/version/backup.go

```diff
@@ -36,6 +36,10 @@ const (
 	// name.
 	OneDrive6NameInMeta = 6
 
+	// OneDrive7MigrateUserPNToID marks when we migrated repo refs from the user's
+	// PrincipalName to their ID for stability.
+	OneDrive7MigrateUserPNToID = 7
```

Comment:
nit: this change isn't just for OneDrive, it's for all services. It's just OneDrive is the only one that needs special handling. Not sure what prefix we should use besides OneDrive though
```suggestion
	All7MigrateUserPNToID = 7
```

Reply:
Yeah, I waffle between naming them for who is checking it, versus who it applies to.  Ah well, it's all fast and loose anyway.